src/client/reducers/table.js

```diff
@@ -0,0 +1,24 @@
+const table = (state = [], action) => {
+  switch (action.type) {
+  case 'CREATE_ROW':
+    return state.concat([action.obj]);
+  case 'DELETE_ROW':
+    state.splice(state.indexOf(action.obj), 1);
+    return [
+      ...state,
+    ];
+  case 'UPDATE_CELL':
+    return state.map((todo, index, arr) => {
```

Comment:
Naming `todo` here has ambiguous meaning.  I suggest to rename the first argument to `row` and refactor your function like following.

```js
cons table = ( ... ) => {
  case 'UPDATE_CELL':
    const { name, value, id } = obj;
    return state.map((row) => { // index and array were redundant actually.
      if (row.seq.toString() === id) {
        return {
          ...row,
          [name]: value,
        }
      }
    })

} 
```